Source/Widgets/Animation/Animation.js

```diff
@@ -698,6 +703,31 @@ define([
      * animation.applyThemeChanges();
      */
     Animation.prototype.applyThemeChanges = function() {
+        // Since we rely on computed styles for themeing, we can't actually
+        // do anything if the container has not yet been added to the DOM.
+        // Set up an observer to be notified when it is added and apply
+        // the changes at that time.
+        if (!document.body.contains(this._container)) {
+            if (defined(this._observer)) {
+                //Already listening.
+                return;
+            }
+            var that = this;
+            that._observer = new MutationObserver(function(mutations) {
+                mutations.forEach(function(mutation) {
+                    for (var i = 0; i < mutation.addedNodes.length; i++) {
+                        if (mutation.addedNodes[i].contains(that._container)) {
```

Comment:
I got it to crash by changing [this line in the TimelineDemo](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Apps/TimelineDemo/TimelineDemo.js#L114) to `animation = new Animation(document.createElement('div'), animationViewModel);`